### PR TITLE
feat: create InputSearch component

### PR DESCRIPTION
**Pull request by kimak**
## Checklist before requesting a review
- [x] Working on iOS
- [x] Working on Android
- [x] Integration tests added
- [x] Visual regressions screenshots are up to date
- [x] Design validated

### CODE REVIEW

File: packages/sandbox/e2e/android/screenshots/InputSearch.png
Comment (kimak) [inline packages/sandbox/e2e/android/screenshots/InputSearch.png:1]: @ianrobertlambert to review on Android

File: packages/sandbox/e2e/ios/screenshots/InputSearch.png
Comment (kimak) [inline packages/sandbox/e2e/ios/screenshots/InputSearch.png:1]: @ianrobertlambert to review on iOS

**Comment by hcourthias** [comment]
We do not have the animated "Cancel" button anymore ? or did you choose to implement it in the project directly ?

**Review by hcourthias — APPROVED** [review]

**Comment by kimak** [comment]
> We do not have the animated "Cancel" button anymore ? or did you choose to implement it in the project directly ?

yes, we keep it separated for better reusability

**Review by judastheo — APPROVED** [review]

**Comment by ianrobertlambert** [comment]
One question:

- Is there 8px of space between the "Search" icon and the text? It looks quite tight on my side.

Other than that, all looks good @kimak ✨🙌

**Pull request merged**